examples/accession2taxid_dag_test.json

```diff
@@ -1,4 +1,5 @@
 {
+  "name": "",
```

Comment:
You can instantiate these with the name of the file, i.e. "accession2taxid_dag_test".